ext/console/02_console.js

```diff
@@ -390,61 +390,85 @@
     const entries = [];
     let iter;
     let valueIsTypedArray = false;
+    let entriesLength;
 
     switch (options.typeName) {
       case "Map":
         iter = MapPrototypeEntries(value);
+        entriesLength = value.size;
         break;
       case "Set":
         iter = SetPrototypeEntries(value);
+        entriesLength = value.size;
```

Comment:
Primordials - `SetPrototypeGetSize`.

Reply:
Didn't see any of `SetPrototypeGetSize`, `TypedArrayPrototypeGetLength` etc used anywhere. I wasn't aware we were using it as well, thought it was only for methods, not properties.
I'll push it in a bit

Reply:
Well, most of the codebase handling sets/maps uses an internal `SafeSet`/`SafeMap`.

But if you spot any of these that aren't being used where you suspect that they should, feel free to open an issue or PR; and maybe I'll try to help